### PR TITLE
add wanda support e2e test on rayci

### PR DESCRIPTION
**Pull request by aslonnie**
testing wanda + rayci on rayci pipeline

### CODE REVIEW

File: .buildkite/hello.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello, Vision!
```
Comment (can-anyscale) [inline .buildkite/hello.txt:1]: lol

File: .buildkite/tests.rayci.yaml [hunk 2]
```diff
@@ -9,4 +9,4 @@ steps:
       - go fmt ./...
       - git diff --exit-code
     queue: small
-    depends_on: forge
+    depends_on: forge
```
Comment (can-anyscale) [inline .buildkite/tests.rayci.yaml:12]: no change

**Review by can-anyscale — APPROVED** [review]

File: .buildkite/tests.rayci.yaml (hunk 2 above)
Comment (aslonnie) [reply .buildkite/tests.rayci.yaml]: get this file reverted

**Pull request merged**